### DISCUSSION TITLE
Adding maize cultivars in APSIM next-gen for America / rest of the world

### DISCUSSION THREAD

**Discussion by ysharaby1102** [APSIMInitiative/ApsimX#9316] [Q&A]
Hi all,
Is there a simple way to add new cultivars to APSIM next-gen? I am trying to run a maize simulation in the US, but none of the ~50 cultivars in the system is currently used in the US, where most of the cultivars are from Australia, New Zealand, or Africa... Is there a database that can be simply downloaded to APSIM next-gen with cultivars from additional countries (US, Canada, Brazil, Europe...)?
Thanks!

**Comment by LenLon** [comment]
I do not know of a database with additional maize cultivars for APSIMx (though if you find one tell me) but it is quite straightforward to add new cultivars to your maize model:

Right-click on the maize model --> Add model --> Cultivar (in the PMF folder)

![grafik](https://github.com/user-attachments/assets/393036d3-f233-40da-8476-598e66a89074)

In the cultivar you can then add your known parameters, a typical cultivar can look like this:

```
[Phenology].Juvenile.Target.FixedValue = 280
[Phenology].Photosensitive.Target.XYPairs.X = 0, 12.5, 24
[Phenology].Photosensitive.Target.XYPairs.Y = 0, 0, 264.5
[Phenology].FlagLeafToFlowering.Target.FixedValue = 10
[Phenology].FloweringToGrainFilling.Target.FixedValue = 170
[Phenology].GrainFilling.Target.FixedValue = 600
[Rachis].DMDemands.Structural.DMDemandFunction.MaximumOrganWt.FixedValue = 14
[Grain].MaximumGrainsPerCob.FixedValue = 360
[Grain].MaximumPotentialGrainSize.FixedValue = 0.217
```

I think you can add all manner of parameters here, though it requires some knowledge of APSIMx' Plant Modelling Framework. Use "Show model structure" on the maize model to take a look!
Rename your cultivar to the desired name if needed, it shows up together with the other cultivars in your sowing manager.
